src/services/ajv.ts

```diff
@@ -49,22 +52,58 @@ export const setDefaultsValidator = (function () {
 
 function setJSONFormDefaults(jsonSchema: any, formData: any) {
     const hydrateAndValidate = setDefaultsValidator.compile(jsonSchema);
+
     hydrateAndValidate(formData);
+
     return hydrateAndValidate;
 }
 
-export function createJSONFormDefaults(jsonSchema: any): {
+function defaultResourceSchema(resourceSchema: any, collection: string) {
```

Comment:
If this is being handled in a store then I think we're good with removing the similar logic in the resource config form.

Reply:
No, that logic cannot be removed as it stands today. Additional logic would need to be added to the store to handle defaulting bindings added to the list post-hydration. Could be another fast follow but I would advocate for it be considered out of scope for this PR. This PR adds a needed UX improvement.

Reply:
Ahhh- yup... that's why it is still here :)

Yeah do not do it in this PR for sure - can be a nice little win later on.